### PR TITLE
Create assets, build, views folders. Load date and time, update every second.

### PR DESCRIPTION
**Pull request by Brian-Lam**
Create assets, build, and views folder for project cleanliness. 
* build: Source files for JS and SASS
* assets: Compiled JS and SASS files
* views: HTML pages

Load date and time into DOM.
Update date and time every second.

### CODE REVIEW

File: build/scripts/main.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
+$(document).ready(function() {
+    UpdateDateTime();
+});
+
+/*
+Load the date and populate the DOM.
+*/
+function LoadDate()
+{
+    // Date
```
Comment (briantotheyanyan) [inline build/scripts/main.js]: Don't need this comment. Doesn't add anything to the understanding

File: build/scripts/main.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    LoadTime();
+    var intervalTime = setInterval(LoadTime, 100);
+}
+
+/*
+Show two digits for minutes and time. 
+
+Source: 
+*/
+function TwoDigitString(_input)
```
Comment (briantotheyanyan) [inline build/scripts/main.js]: Nit: I might rename this function to make it more explicit what it does.

**Review by briantotheyanyan — APPROVED** [review]

File: build/scripts/main.js (hunk 2 above)
Comment (Brian-Lam) [reply build/scripts/main.js]: Fair enough.

File: build/scripts/main.js (hunk 1 above)
Comment (Brian-Lam) [reply build/scripts/main.js]: Yeap, leftover comment from refactoring

**Pull request merged**